src/ecdsa/ellipticcurve.py

```diff
@@ -1284,6 +1284,8 @@ def __init__(self, curve, x, y, z, t, order=None):
         else:  # pragma: no branch
             self.__coords = (x, y, z, t)
             self.__order = order
+        self.__generator = generator
+        self.__precompute = []
```

Comment:
because of `_maybe_precompute()`? the precomputation is delayed until the first time multiplication is performed, so `__generator == True` means "yes, we want to precompute multiplication table", while non empty `__precompute` means we actually did precompute

Reply:
Oh, right. Guess a tri-state value isn't a particularly good idea either.